server_addon/applications/client/ayon_applications/addon.py

```diff
@@ -171,3 +202,17 @@ def _cli_extract_environments(
 
         with open(output_json_path, "w") as file_stream:
             json.dump(env, file_stream, indent=4)
+
+    def _cli_launch_applications(self, project, folder, task, app):
+        """Produces json file with environment based on project and app.
+
+        Called by farm integration to propagate environment into farm jobs.
```

Comment:
Pretty sure this docstring is incorrect.

Reply:
Copy pasta.